tests/seeds/creds.js

```diff
@@ -0,0 +1,22 @@
+const {generate} = require('../../server/auth/hash')
```

Comment:
let's delete this file

Reply:
Done 6700a6d300569ac2cc2413e24c6cf1b22400302f